src/mbgl/programs/symbol_program.hpp

```diff
@@ -94,7 +94,7 @@ class SymbolIconProgram : public Program<
                                        const TransformState&);
 };
 
-class SymbolSDFProgram : public Program<
+class SymbolSDFIconProgram : public Program<
```

Comment:
Yeah, that's something like what I was trying before and having trouble with -- but it's very possible that it was just other mid-refactor issues that were making it hard to see clearly.  Lemme try again.